esmvalcore/cmor/_fixes/cmip6/fgoals_g3.py

```diff
@@ -39,3 +40,45 @@ def fix_metadata(self, cubes):
 
 
 Siconc = Tos
+
+
+class Mrsos(Fix):
+    """Fixes for mrsos."""
+
+    def _check_bounds_monotonicity(self, coord):
+        """Check monotonicity of a coords bounds array."""
+        if coord.has_bounds():
+            for i in range(coord.nbounds):
+                if not iris.util.monotonic(coord.bounds[..., i], strict=True):
+                    return False
```

Comment:
you are using `self` as functional argument but it looks like you are not using it in the function itself, if this is a stand alone function and not a class instance, please put it outside the class

Reply:
No problem. Done.

Reply:
cheers :beer: